servers.json

```diff
@@ -1370,5 +1370,12 @@
         "addresses": [
             "dropthem.eu"
         ]
+    },
+    {
+        "id": "heavenpvp",
```

Comment:
Add your server logo in the `logos` directory

Reply:
It won't let me upload my logo to the logo directory

Reply:
https://github.com/Santy02/ServerMappings/blob/master/logos/heavenpvp.png

Reply:
![heavenpvp](https://user-images.githubusercontent.com/75143460/168447865-f38bf577-20dc-468e-b5c7-de4a837b5239.png)

Reply:
It won't let me upload my logo to the praise directory

Reply:
You upload it to your fork not this repo

Reply:
https://github.com/Santy02/ServerMappings/blob/master/logos/heavenpvp.png